package.json

```diff
@@ -74,18 +74,17 @@
   "devDependencies": {
     "ajv": "^6.12.0",
     "babel-cli": "^6.26.0",
-    "babel-eslint": "^10.0.3",
```

Comment:
Forgot to remove this when upgrading to `eslint-config-skyscanner@5.0.0` in https://github.com/sogame/danger-plugin-toolbox/pull/247